geniza/corpus/views.py

```diff
@@ -810,6 +829,14 @@ def get_context_data(self, **kwargs):
             # transcription always rtl
             text_direction = "rtl"
 
+        # extended show default/disabled logic from document detail view
```

Comment:
This comment isn't quite clear enough to easily understand what's going on. You're overriding the defaults for transcription based on which thing they are editing?

I think it should be possible to simplify these list comprehensions — hopefully in the main detail view to simplify this code.